tests/test_vnet.py

```diff
@@ -790,7 +902,7 @@ def test_vnet_orch_1(self, dvs, testlog):
         vnet_obj.check_del_vnet_routes(dvs, 'Vnet_2001')
 
         delete_vnet_routes(dvs, "100.100.1.1/32", 'Vnet_2000')
-        vnet_obj.check_del_vnet_routes(dvs, 'Vnet_2001')
+        vnet_obj.check_del_vnet_routes(dvs, 'Vnet_2000')
```

Comment:
Why is this changed?

Reply:
I still believe this seems more like a typo, though I do not recall how I noticed this (maybe I added some more checks in the function and the test case failed). As we just removed a route from `Vnet_2000`, why do we want to check deleted route in `Vnet_2001`? I checked other calls of the same function and nowhere else use it in this way. I think it passed previously only because `check_del_vnet_routes` did not really check anything.

Reply:
ok